### ISSUE TITLE
Enable nf-api to be exposed through NodePorts

### ISSUE THREAD

**Issue by cbmckni** [SciDAS/nextflow-api#37]
For submitting jobs on TACC's Rodeo cluster, we need to use NodePorts to expose nextfow-api.

This will involve exposing both the nextflow-api server and the MongoDB server.

```
When you create services, use NodePort like so:

apiVersion: v1
kind: Service
metadata:  
  name: postgres
spec:
  selector:    
    app: postgres
  type: NodePort
  ports:  
  - name: postgres
    port: 5432
    targetPort: 5432

Then you'll get a randomly assigned port that you can reach on 172.16.120.49. e.g. `kubectl get service` :

postgres            NodePort    10.96.221.153   <none>        3000:31888/TCP   3h2m
```

**Comment by cbmckni** [comment]
Pushed initial attempt to https://github.com/SciDAS/nextflow-api/tree/37-enable-node-ports

@bentsherman Does Mongo need to be exposed to the internet? May not even need a nodeport.

**Comment by bentsherman** [comment]
@cbmckni The database pod doesn't need to be exposed to the internet. Not sure if I ever told you this or not.

**Comment by cbmckni** [comment]
Fixed with f2fa5a35087d62fa31705429ba44c0d8f8fbd2a4
